src/libraries/Common/tests/TestUtilities/System/SecretsHelper.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class SecretsHelper
+    {
+        public static string RandomPassword()
+        {
+            return Guid.NewGuid().ToString();
+        }
+
+        // Explicitly mark password as it is not a secret.
```

Comment:
Can we be more specific in the comment? Or it would go against the whole secrecy around the super secrete internal tool :smile: